README .R

```diff
@@ -0,0 +1,28 @@
+
+library(dplyr)
+library(readr)
+library(stringr)
+library(sbtools)
+library(whisker)
+
+source("1_fetch/src/fetch_data.R")
+source("2_process/src/process_data.R") 
+source("3_visualize/src/visualize_data.R")
+
+project_output_dir <- '1_fetch/out'
+mendota_file <- file.path(project_output_dir, 'model_RMSEs.csv')
+
+fetch_data(mendota_file)
+
+
+
+project_output_dir <- '2_process/out'
+eval_data <- load_data(mendota_file = mendota_file)
+summary_model(mendota_file = eval_data)
```

Comment:
I don't love this name ('mendota_file') for the argument that gets passed into `summary_model`. I think `summary_model()` needs a data.frame (from R's memory), right? Not a file (which would be saved as a file in your file system). You can choose a name that makes sense to you, but I personally would go with something more like "eval_data", "model_data", "eval_data_frame", "eval_df", etc.

Reply:
this comment still stands (in review number 2)